### DISCUSSION TITLE
Github Actions are broken in my project

### DISCUSSION THREAD

**Discussion by RouHim** [community/community#22047] [New to GitHub]
<p>Hi github community,<br>
I’ve migrated a project from gitlab to github: <a href="https://github.com/RouHim/disCoverJ" class="inline-onebox" rel="noopener nofollow ugc">GitHub - RouHim/disCoverJ</a><br>
As one of the first things, I wanted to setup a github actions pipeline:<br>
<a href="https://github.com/RouHim/disCoverJ/blob/778364f229fe8d0e3edd8231861022f718ac8117/.github/workflows/main.yml" class="inline-onebox" rel="noopener nofollow ugc">disCoverJ/main.yml at 778364f229fe8d0e3edd8231861022f718ac8117 · RouHim/disCoverJ · GitHub</a></p>
<p><strong>My Problem:</strong><br>
All my pipe runs fail with the following error:</p>
<blockquote>
<p>An unexpected error has occurred and we’ve been automatically notified. Errors are sometimes temporary, so please try again.</p>
</blockquote>
<p>E.g.: <a href="https://github.com/RouHim/disCoverJ/actions/runs/2014452255" class="inline-onebox" rel="noopener nofollow ugc">Update main.yml · RouHim/disCoverJ@68cf893 · GitHub</a></p>
<p>I have been getting the error since last night, so overnight the problem did not go away.<br>
Does anyone have any idea what the problem is?<br>
I tried to run the pipe only with the checkout action, that didn’t work either.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>In that case it’s probably best to contact support directly so they can check. The link is error message, and make sure to include the error ID mentioned there. Good luck!</p>

**Comment by inesverissimodnxt** [comment]
Hi @RouHim 
Can you please share the solution? 
Thank you
